<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editContact</class>
 <widget class="QWidget" name="editContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>357</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>427</width>
    <height>357</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>427</width>
    <height>357</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>edit contact window</string>
  </property>
  <widget class="QLabel" name="note_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>заметка</string>
   </property>
  </widget>
  <widget class="QLabel" name="photo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {background-color: rgb(205, 205, 205)};</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>401</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>применить изменения</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="name_label">
        <property name="text">
         <string>имя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="surname_label">
        <property name="text">
         <string>фамилия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="phone_label">
        <property name="text">
         <string>телефон</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="email_label">
        <property name="text">
         <string>почта</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="adress_label">
        <property name="text">
         <string>адрес</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="name_edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="surname_edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="phone_edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="email_edit"/>
      </item>
      <item>
       <widget class="QTextEdit" name="adress_edit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="set_image">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>выбрать фото</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="note_edit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
